theme/templates/pages/features.html

```diff
@@ -117,7 +117,7 @@ <h2><a class="headline-link" href="#samplers">{% trans %}Sampler Decks{% endtran
 <div id="effects" class="docs-section container">
   <h2><a class="headline-link" href="#effects">{% trans %}Effects Chains{% endtrans %}</a></h2>
   <p>
-    {% trans %}Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob.{% endtrans %}
+    {% trans %}Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob. Rearrange and hide parameters, and set defaults. Export your custom effect chains to share it with others.{% endtrans %}
```

Comment:
lets also mention the availability from the mixer, something like this?
```suggestion
    {% trans %}Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob. Rearrange and hide parameters, and set defaults. Export your custom effect chains to share it with others. Assign your custom chains to the Filter/QuickEffect knob to add color right from your mixer controls.{% endtrans %}
```

Reply:
Yup!